spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/web/servlet/WebMvcMetricsFilter.java

```diff
@@ -88,10 +92,31 @@ public WebMvcMetricsFilter(ApplicationContext context, MeterRegistry registry,
 	 */
 	public WebMvcMetricsFilter(MeterRegistry registry, WebMvcTagsProvider tagsProvider,
```

Comment:
I think we should deprecate this constructor in favour of the new one added below.